build.sbt

```diff
@@ -117,6 +117,29 @@ lazy val root = (project in file("."))
     )
   )
 
+lazy val scalaVersionDependentSettings = Def.settings(
+  libraryDependencies ++= (if (scalaVersion.value.startsWith("2."))
```

Comment:
Could you please start the `if` in the next line so the block doesn't get indented too far right?

Reply:
Done 👍